source/text/Format.h

```diff
@@ -13,6 +13,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #ifndef ES_TEXT_FORMAT_H_
 #define ES_TEXT_FORMAT_H_
 
+#include <array>
```

Comment:
Ah, was going by `#include <vector>` being in the header file, as well.

Reply:
Yeah, `map` & `vector` are necessary here because they are return values or by-value parameter types used in this class's method signatures. Unlike them, `std::array` is used as an implementation detail and doesn't need to be advertised to other classes.